openmp/runtime/src/kmp_wait_release.h

```diff
@@ -1038,15 +1038,13 @@ static inline void __kmp_null_resume_wrapper(kmp_info_t *thr) {
   case flag_oncore:
     __kmp_resume_oncore(gtid, RCAST(kmp_flag_oncore *, flag));
     break;
-#ifdef KMP_DEBUG
```

Comment:
If these were kept out for performance reasons, maybe we should just put an empty default in an `#else` branch? (I don't know this code though, just speculating..)

Reply:
Changed this a bit, can you take another look please?